### PR TITLE
ui: use route location instead of window.location (PROJQUAY-5392)

### PR DESCRIPTION
**Pull request by syed**
window.location causes issues when running in beta or preview on console

### CODE REVIEW

File: web/src/routes/RepositoriesList/RepositoriesList.tsx [hunk 1]
```diff
@@ -69,6 +69,9 @@ export default function RepositoriesList(props: RepositoriesListProps) {
   const [makePublicModalOpen, setmakePublicModal] = useState(false);
   const [makePrivateModalOpen, setmakePrivateModal] = useState(false);
   const [err, setErr] = useState<string[]>();
+  const location = useLocation();
+
+  console.log('current route location', location.pathname);
```
Comment (Sunandadadi) [inline web/src/routes/RepositoriesList/RepositoriesList.tsx:74]: Can you remove the console.

File: web/src/routes/RepositoriesList/RepositoriesList.tsx [hunk 2]
```diff
@@ -414,3 +429,7 @@ interface RepoListTableItem {
   size: number;
   last_modified: number;
 }
+
+interface RepositoriesListProps {
+  organizationName: string;
```
Comment (Sunandadadi) [inline web/src/routes/RepositoriesList/RepositoriesList.tsx:434]: Is there a reason for adding the props here? I don't see the `organizationName` getting used in the PR.

File: web/src/routes/TagDetails/TagDetailsTabs.tsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ function getTabIndex(tab: string) {
 export default function TagTabs(props: TagTabsProps) {
   const [activeTabKey, setActiveTabKey] = useState<TabIndex>(TabIndex.Details);
   const navigate = useNavigate();
+  const location = useLocation();
```
Comment (Sunandadadi) [inline web/src/routes/TagDetails/TagDetailsTabs.tsx:23]: `location` here doesn't seem to getting used in the PR.

File: web/src/routes/RepositoriesList/RepositoriesList.tsx (hunk 2 above)
Comment (syed) [reply web/src/routes/RepositoriesList/RepositoriesList.tsx:434]: I see `RepositoriesListProps` used in `src/routes/RepositoriesList/RepositoriesList.tsx:65` but it's not defined so I'm defining it

File: web/src/routes/RepositoriesList/RepositoriesList.tsx (hunk 1 above)
Comment (syed) [reply web/src/routes/RepositoriesList/RepositoriesList.tsx:74]: removed

File: web/src/routes/TagDetails/TagDetailsTabs.tsx (hunk 3 above)
Comment (syed) [reply web/src/routes/TagDetails/TagDetailsTabs.tsx:23]: it's used in `src/routes/TagDetails/TagDetailsTabs.tsx:35` but not defined in the context which causes issues

**Review by Sunandadadi — APPROVED** [review]
lgtm

**Pull request merged**